contents/docs/data/events.mdx

```diff
@@ -70,3 +70,52 @@ If something has been detected incorrectly, you can manually change the type by
     alt="Edit properties" 
     classes="rounded"
 />
+
+### Default properties
```

Comment:
This is used here, in the [autocapture docs](https://posthog.com/docs/product-analytics/autocapture#autocaptured-properties), and in the tutorial. 

I thought about creating a snippet but decided not two:
- Can't use snippets in tutorials
- Snippets are a bit of a pain to work with, so didnt feel worth it for these docs tables, since we reference the github anyways

Reply:
Oh interesting, what's the story behind snippets and tutorials?

Reply:
If you make the tutorial an `.mdx` shouldn't the snippets work?